share/templates/08-Orchestral/01-Classical_Orchestra.mscx

```diff
@@ -63,6 +66,7 @@
         </section>
       <section id="brass" brackets="true" showSystemMarkings="false" barLineSpan="true" thinBrackets="true">
         <family>horns</family>
+        <family>flugelhorns</family>
```

Comment:
Please make a similar change in the <code>orchestral</code> score order in <code>orders.xml</code> (or remove it here, as long both orders are the same).